RELEASE_NOTES/3.54.md

```diff
@@ -0,0 +1,20 @@
+# CLI
+* ***[`oclif` upgrade.](https://github.com/Shopify/cli/pull/2975)*** Shopify CLI is built on [oclif](https://oclif.io/).
+v3 comes with a number of user-facing improvements, such as better validations and a new style for help text.
+
+# App
+* ***[Sync GraphiQL header bar links.](https://github.com/Shopify/cli/pull/3232)*** The link pills in the header bar will
+now be updated with the correct app and store when `app dev` is restarted.
+* ***[Fix: Attempt auth refresh when updating extension drafts.](https://github.com/Shopify/cli/pull/3233)*** Rather than
+failing to update extensions, the CLI will refresh authentication and reattempt.
+* ***[Fix: Postinstall step for Cloudflare.](https://github.com/Shopify/cli/pull/3288)*** When `cloudflared` is
+installed but doesn't work, an attempt will be made to reinstall.
+
+# Theme
+* ***[New Theme Command: `shopify theme rename`.](https://github.com/Shopify/cli/pull/3242)*** The command renames an
+existing theme.
+* ***[`shopify theme pull` without Ruby.](https://github.com/Shopify/cli/pull/3139)*** `theme pull` is the next command
+to be rewritten to work with Node only.
```

Comment:
Fair enough. It's the kind of thing we'd mention on the changelog (in case anything went wrong) but not in these notes because it's not a "selling point" of the new version.

It will be a big deal when we're Ruby-free, but we're not there yet.